src/Model/Resolver/DataProvider/Page.php

```diff
@@ -0,0 +1,75 @@
+<?php
+/**
+ * ScandiPWA_CmsGraphQl
+ *
+ * @category    Scandiweb
+ * @package     ScandiPWA_CmsGraphQl
+ * @copyright   Copyright (c) 2018 Scandiweb, Ltd (https://scandiweb.com)
+ */
+
+declare(strict_types=1);
+
+namespace ScandiPWA\CmsGraphQl\Model\Resolver\DataProvider;
+
+use Magento\Cms\Api\Data\PageInterface;
+use Magento\Cms\Api\PageRepositoryInterface;
+use Magento\Framework\Exception\NoSuchEntityException;
+use Magento\Widget\Model\Template\FilterEmulate;
+
+/**
+ * Cms page data provider
+ */
+class Page extends \Magento\CmsGraphQl\Model\Resolver\DataProvider\Page
+{
+    /**
+     * @var FilterEmulate
+     */
+    private $widgetFilter;
+
+    /**
+     * @var PageRepositoryInterface
+     */
+    private $pageRepository;
+
+    /**
+     * @param PageRepositoryInterface $pageRepository
+     * @param FilterEmulate $widgetFilter
+     */
+    public function __construct(
+        PageRepositoryInterface $pageRepository,
+        FilterEmulate $widgetFilter
+    ) {
+        $this->pageRepository = $pageRepository;
+        $this->widgetFilter = $widgetFilter;
+    }
+
+    /**
+     * @param int $pageId
+     * @return array
+     * @throws NoSuchEntityException
+     */
+    public function getData(int $pageId): array
+    {
+        $page = $this->pageRepository->getById($pageId);
+
+        if (false === $page->isActive()) {
```

Comment:
Please don't use yoda style, prefer left-to-right instead: $page->isActive() === false, or even simply **if (!$page->isActive())**